official/resnet/cifar10_test.py

```diff
@@ -42,6 +42,7 @@ class BaseTest(tf.test.TestCase):
   @classmethod
   def setUpClass(cls):  # pylint: disable=invalid-name
     super(BaseTest, cls).setUpClass()
+    tf.compat.v1.disable_eager_execution()
```

Comment:
do you need to check "if 2.0" before this? not sure because you check in some places not others, and i wasn;t sure if that is intentional or not